docs/specification/draft/server/profiles.mdx

```diff
@@ -0,0 +1,137 @@
+  ```
+  https://modelcontextprotocol.io/specification/<version>/server/standard-profiles/
+  ```
+- Any profiles available outside of the standard profiles location are called Custom Profiles.
+- MCP Servers may simultaneously support a mix of standard and custom profiles.
+
+### Profile Modification and Versioning
+Once made available, Profiles MUST NOT be changed. In order to make changes to a Profile, the author MUST create a new version of the Profile, with its own Profile URL.
+
+## Publishing Profiles
+An MCP Server makes it possible for Clients to discover the profiles it supports as follows:
+
+### ProfileSpec Object
+
+A ProfileSpec object has the following fields:
+
+- The location of the profile specification
+- The minimum MCP version required to use that profile.
+
+```TypeScript
+interface ProfileSpec {
+  "profileURL": string; 
+  // The unique URL identifying the profile,
+  // which is also location where the profile is specified.
+
+  "minMcpVersion": string;
+  // this is the date in YYYY-MM-DD format that identifies the MCP version.
+}
+```
+
+### Profiles Declaration
+The profiles supported by an MCP Server are available through the Profiles Declaration JSON format document that is an array of `ProfileSpec` objects:
+
+```TypeScript
+[ ProfileSpec, ProfileSpec, ...]
+```
+
+#### Default Profile
+A default profile is one that the server uses if the client does not request a specific profile. The first `ProfileSpec` in the Profiles Declaratoin document is the Server's default profile.
+
+### Well-known URL
+The Profiles Declaration is made available by an MCP Server at its "well-known" location, as described in [RFC8615](https://datatracker.ietf.org/doc/html/rfc8615). The file name for the MCP Profiles Declaration document is: `mcp-profiles`. Thus the well-known URL for the MCP Profiles Declaration is obtained by inserting the path:
+
+```
+/.well-known/mcp-profiles
+```
+
+betewen the "authority" component (i.e. the domain name or IP address and port) and any path component of the MCP Server's URL.
+
+So for example, if your MCP server is located at: `https://mymcpserver.mycompany.com`, then the profiles declaration document is at: `https://mymcpserver.mycompany.com/.well-known/mcp-profiles`. If your MCP server is located at: `https://mycompany.com/mymcpserver`, then the profiles declaration document is at: `https://mycompany.com/.well-known/mcp-profiles/mymcpserver`.
+
+### Backward compatibility
+If this resource does not exist on the MCP Server, or if the document returned is empty, then the Client MUST assume that the MCP Server does not support Profiles.
+
+## Profile Discovery
+Before connecting to an MCP Server, an MCP Client MAY fetch the Server’s Well Known URI to understand which profiles the server supports (if any).
+
+## Initialization With Profiles
+The MCP Client makes an `InitializeRequest` based on its knowledge of which (if any) profiles the MCP Server supports.
```

Comment:
Way too into the implementation details for a generic spec. You can mention that profiles need to be obtained by the client before initiating any interaction requests, or something along those lines.